README.md

```diff
@@ -1 +1,3 @@
-# handle_locator
\ No newline at end of file
+# handle_locator
+
+[![CI](https://github.com/tue-robotics/handle_locator/workflows/CI/badge.svg)](https://github.com/tue-robotics/handle_locator/actions)
```

Comment:
Please expand on the readme. Al least give a brief description of what belongs in this repository